src/configuration/configuration.ts

```diff
@@ -240,6 +240,8 @@ class Configuration implements IConfiguration {
 
   surround = true;
 
+  disableDefaultPluginMappings = false;
+
```

Comment:
about this switch: I want to do one switch for all cases (right now there is only surround. there might be coerce cr or exchange cx).
Not one per possible plugin; that would bloat the code and config.
the only real use case I see, is the few users with custom keyboard layouts. For that, mappings like cr or cx will be wrong same as cs.